Makefile

```diff
@@ -259,7 +259,7 @@ update-contributors: check-github-token
 	@echo "Updating CONTRIBUTORS.md..."
 	@python maintainers/fetch_contributors.py \
 		--repo AcademySoftwareFoundation/OpenTimelineIO \
-		--token $(OTIO_RELEASE_GITHUB_TOKEN)
+		--token "${OTIO_RELEASE_GITHUB_TOKEN}"
```

Comment:
are you sure about this change?  The `$(OTIO_RELEASE_GITHUB_TOKEN)` means the variable is being subsituted in make, but the extra quoting and braces mean its going to be evaluated in the subshell...I'm not sure if there is a difference.  Is that because you don't want the token to appear in the shell output?

Reply:
I'll admit, I'm not sure about this. It seemed like it'd fixed an issue with the token not coming through properly. I'm going to revert this change though and I'll see if everything goes through okay next release without it.